<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../recursos.qrc">
    <normaloff>:/src/vista/imagenes/logoUniConecta.png</normaloff>:/src/vista/imagenes/logoUniConecta.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background-color: #f0f4f1;
    }
    QLabel {
        font: 20pt &quot;Segoe UI&quot;;
        color: #2e7d32;
    }
    QPushButton {
        background-color: #2e7d32;
        color: white;
        border-radius: 10px;
        padding: 6px 12px;
        font: bold 10pt &quot;Segoe UI&quot;;
    }
    QPushButton:hover {
        background-color: #388e3c;
    }
    QLineEdit {
        background-color: transparent;
        border: none;
        border-bottom: 3px solid #2e7d32;
        font: 10pt &quot;Segoe UI&quot;;
        padding: 4px;
    }
    QLineEdit:focus {
        border-bottom: 3px solid #66bb6a;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Century&quot;;</string>
    </property>
    <property name="text">
     <string>Iniciar Sesión</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonAceptar">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>93</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: rgb(0, 85, 0);
        color: white;
        border-radius: 8px;
        font: 10pt &quot;Segoe UI&quot;;
      }
      QPushButton:hover {
        background-color: #388e3c;
      }</string>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonVolver">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>93</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: rgb(0, 85, 0);
        color: white;
        border-radius: 8px;
        font: 10pt &quot;Segoe UI&quot;;
      }
      QPushButton:hover {
        background-color: #388e3c;
      }</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BotonRecuperar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        background-color: rgb(0, 85, 0);
        color: white;
        border-radius: 8px;
        font: 10pt &quot;Segoe UI&quot;;
      }
      QPushButton:hover {
        background-color: #388e3c;
      }</string>
    </property>
    <property name="text">
     <string>He olvidado la contraseña</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="iniciarSesion_correo">
       <property name="placeholderText">
        <string>Correo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="iniciarSesion_contrasena">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Contraseña</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
